gradio/interface.py

```diff
@@ -178,27 +179,19 @@ def __init__(
         if not isinstance(outputs, list):
             outputs = [outputs]
 
+        if "state" in inputs or "state" in outputs:
+            if len([i for i in inputs if i == "state"]) != 1 or len([o for o in outputs if o == "state"]) != 1:
```

Comment:
shall we break this check for readibility?

Reply:
Done.